src/main/java/org/apache/sling/event/impl/EventingThreadPool.java

```diff
@@ -53,11 +55,19 @@ public class EventingThreadPool implements ThreadPool {
     public @interface Config {
 
         @AttributeDefinition(name = "Pool Size",
-              description="The size of the thread pool. This pool is used to execute jobs and therefore "
-                        + "limits the maximum number of jobs executed in parallel.")
-        int minPoolSize() default 35;
+                description = "The size of the thread pool. This pool is used to execute jobs and therefore "
+                        + "limits the maximum number of jobs executed in parallel. "
+                        + "A value of -1 is substituted with the number of available processors. "
+                        + "A decimal number between 0.0 and 1.0 is treated as a fraction of available processors. "
+                        + "For example 0.5 means half of the available processors. ")
+        double minPoolSize() default 35;
```

Comment:
I think that this has the chance to break a system where this value is defined as int. Have you checked how a running works if you provide an int? Is it automatically converted to a double?

Reply:
It is according to https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#service.component-component.property.value.coercion.

Reply:
Yep, just tested it out and it works: 
![Screenshot 2024-10-28 at 14 56 21](https://github.com/user-attachments/assets/a68bab57-7a68-47d1-a9c6-e1d4c2e9fc96)
![Screenshot 2024-10-28 at 14 56 15](https://github.com/user-attachments/assets/970c95e2-c458-4ffb-9a50-ec782c8b280c)
![Screenshot 2024-10-28 at 14 56 27](https://github.com/user-attachments/assets/bfb7dc9a-3af9-4c10-b5fe-5ba5ef3f3152)